staging/src/k8s.io/code-generator/cmd/validation-gen/validators/format_test.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validators
+
+import "testing"
+
+func TestParseFormat(t *testing.T) {
```

Comment:
have a preference toward consolidating tag grammar into a single parser, if at all possible.
 
 I would live with this, but I'd be slightly happier if we could somehow fit this into a system wide grammar, rather than adding tag specific grammar.